src/compile/header/index.ts

```diff
@@ -87,11 +95,23 @@ export function assembleTitleGroup(model: Model, channel: FacetChannel) {
       offset: 10,
       ...(channel === 'row' ? {orient: 'left'} : {}),
       style: 'guide-title',
+      ...titleAlign(titleAnchor),
       ...getHeaderProperties(config, facetFieldDef, HEADER_TITLE_PROPERTIES, HEADER_TITLE_PROPERTIES_MAP)
     }
   };
 }
 
+export function titleAlign(titleAnchor: TitleAnchor) {
+  switch (titleAnchor) {
+    case 'start':
+      return {align: 'left'};
+    case 'end':
+      return {align: 'right'};
+  }
+  // TODO: take TitleAngle into account for the "middle" case
```

Comment:
Why not do this?

Reply:
Much more testing, beyond the scope of this PR.

Reply:
Okay